ci/test/cargo-test/image/run-tests

```diff
@@ -20,7 +20,30 @@ ci_init
 
 while read -r binary cwd; do
     cd "$cwd"
-    ci_try "/tests/$binary" --nocapture
+    ci_try "/tests/$binary" --nocapture -Z unstable-options --format json --report-time | tee results.json
+    ci_try "/usr/local/cargo2junit" < results.json > junit.xml
+    ci_try "/bin/cat" junit.xml | sed 's/"/\\"/g' > escaped.xml
+    junit=$(<escaped.xml)
+    curl --request POST \
+         --url https://analytics-api.buildkite.com/v1/uploads \
+         --header 'Authorization: Token token="9uPd3ZeKfYQ2cKkEyGo1vaPU";' \
```

Comment:
Eep! Probably don't want to be publicizing this.

Reply:
Doh, yeah I didn't intend to commit that!